src/lib/components/molecules/canvas-map/Map.jsx

```diff
@@ -3,16 +3,27 @@ import { forwardRef } from "preact/compat"
 import { Map as _Map } from "./lib/Map"
 import { View } from "./lib/View"
 import styles from "./style.module.scss"
+import { MapProvider } from "./context/MapContext"
 
 const mobileHelpText = "Use two fingers to zoom"
 
+/**
+ * Map component that renders into a canvas.
```

Comment:
Not very helpful documentation here, sorry! I was thinking we could do a wave of proper documentation for the most important components a bit later on?